lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
 
     if (command && command === '/ramal') {
-        const findExtension = text ?
-            extCommand.find.bind(extCommand, text) :
-            extCommand.find.bind(extCommand);
-
-        findExtension(out => {
-            res.send(out);
-        });
+        if (update && update === 'true' && config.extMode === 'SHEET') {
+            extCommand.update(text, out => {
```

Comment:
I thought it would be cool if a user could make an update request and make a search in the same request so i went the extra mile to make it work. But if there's a real technical obstacle in slack's side for it to work I can change the implementation and separate concerns. Otherwise I think we should keep it how it is.